lib/nsxv.py

```diff
@@ -326,14 +329,25 @@ def __str__(self):
     for s in services:
       service = '%s%s' % (service, s)
 
+    #applied To
+    appliedToList = ''
```

Comment:
This is an invalid name for a variable. Please see the python style guide we try to maintain.
https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

You will want to fix this for appliedToList and appliedTo